container/container.go

```diff
@@ -14,6 +14,12 @@ import (
 	"github.com/nspcc-dev/neofs-sdk-go/version"
 )
 
+// Container is a structure that defines object placement behaviour.
+//
+// Container is mutually compatible with github.com/nspcc-dev/neofs-api-go/v2/container.Container
+// message. See ReadFromV2 / WriteToV2 methods.
+//
+// Instances must be created via New function.
```

Comment:
Too strong requirement, instances can be created using `var` declaration. For some scenarios we can provide analogue of `object.InitCreation`.

Reply:
so you mean deleting `New` func or mentioning that var declaration is also ok?

Reply:
Yes, we should refuse `New` functions and provide types with instances which can be safely instantiated using `var`. If there are some business logic like `each container must have owner` we implement it as a function like `InitCreation`.

Reply:
ok, do not really see procs of the `InitCreation` function like in the `obj` service so fixed the comment and *just rename* `New` to `InitCreation` (do not get the idea of creation obj/cnr with any fields(1 TB payload, any address, any signature, etc.) pass it to `InitCreation` func and... expecting what?)

Reply:
https://github.com/nspcc-dev/neofs-sdk-go/blob/f18b4a2a756bc49f26bde1bf202fc25c3daf2413/object/object.go#L19

https://github.com/nspcc-dev/neofs-sdk-go/blob/f18b4a2a756bc49f26bde1bf202fc25c3daf2413/object/object.go#L35

Reply:
so we leave it as it is done no, yes?

my idea was to make an exception for such func (do not change passed object but return *really* blank object from the func since we can control that). but I do not insist